src/rnn/attention.h

```diff
@@ -29,10 +31,24 @@ class GlobalAttention : public CellInput {
   Expr dropMaskState_;
 
   // for Nematus-style layer normalization
-  Expr Wc_att_lns_, Wc_att_lnb_;
-  Expr W_comb_att_lns_, W_comb_att_lnb_;
+  std::vector<Expr> Wc_att_lnss_, Wc_att_lnbs_;
+  std::vector<Expr> W_comb_att_lnss_, W_comb_att_lnbs_;
   bool nematusNorm_;
 
+  int numAttentionHeads_;
```

Comment:
This is a great place to add comments for each of these options to explain what they do. A brief line-end comment for each is sufficient.